frontend-implicaction/src/app/core/services/api-endpoints.service.ts

```diff
@@ -150,6 +150,15 @@ export class ApiEndpointsService {
     return ApiEndpointsService.createUrlWithPathVariables(Uris.RELATIONS.REQUEST, [receiverId]);
   }
 
+  getAllGroups(userId: string): string {
+    const uri = ApiEndpointsService.createUrlWithPathVariables(Uris.USERS.GROUP_LIST, [userId, 'groups'])
+      // createUrlWithPathVariables et createUrlWithPageable ajoutent le endpoint ('/api/') de l'api en début de l'adresse générée
+      // on se retrouve donc avec une répétition en les chaînant. Il faut donc en supprimer un
+      .replace(`${Constants.API_ENDPOINT}/`, '');
+
+    return ApiEndpointsService.createUrl(uri);
```

Comment:
```suggestion
    return ApiEndpointsService.createUrlWithPathVariables(Uris.USERS.GROUP_LIST, [userId, 'groups']);
```

je pense que ça devrait fonctionner si tu mets juste ça